styles.css

```diff
@@ -0,0 +1,176 @@
+body {
+  font-family: "Josefin Sans";
+  background-color: #fffafb;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+.instruction {
+  position: absolute;
+  left: 100px;
+}
+
+.logo img {
+  width: 15%;
+}
+header {
+  position: fixed;
+  background-color: #f4f2f2;
+  top: 0;
+  height: 100px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  min-height: 75px;
+  padding: 0px 20px;
+  font-size: 20px;
+}
```

Comment:
I suggest you give space before writing the next selector. This will make your code more readable 😊 .